package.json

```diff
@@ -17,11 +17,15 @@
     "env": "printenv",
     "clean:dist": "git checkout -- dist",
     "clean": "rm -rf generated dist lib .coverage && yarn run clean:dist",
-    "compile:node": "coffee --output lib --compile src",
-    "compile:test": "coffee --output generated/test --compile test",
+    "compile:node:coffee": "coffee --output lib --compile src",
```

Comment:
yeah but npm sets it. yarn doesn't. yarn sets some $npm_package_* vars but not those